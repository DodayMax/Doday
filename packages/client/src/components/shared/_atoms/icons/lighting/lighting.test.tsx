import * as React from 'react';
import { shallow } from 'enzyme';
import { Icons } from '@shared';

describe('Lighting', () => {
  it('renders correctly without props', () => {
    const lighting = shallow(<Icons.Lighting />);

    expect(lighting).toMatchSnapshot();
  });
});
