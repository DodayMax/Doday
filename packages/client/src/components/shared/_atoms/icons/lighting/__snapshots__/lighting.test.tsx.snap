// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Lighting renders correctly without props 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <path
        d="M17.657 10.333a1.317 1.317 0 0 0-1.1-.6h-2.64V3.511c0-.634-.44-1.18-1.055-1.308a1.317 1.317 0 0 0-1.48.784l-3.96 9.333c-.174.412-.132.885.112 1.26.245.373.66.598 1.103.598h2.64V20.4c-.001.634.44 1.18 1.055 1.308.615.127 1.233-.2 1.48-.784l3.96-9.333c.173-.412.13-.884-.115-1.258zM12.597 20.4v-7.556h-3.96l3.96-9.333v7.556h3.96l-3.96 9.333z"
        fill="black"
        fillRule="evenodd"
      />,
      "height": 20,
      "viewBox": "0 0 24 24",
      "width": 20,
      "xmlns": "http://www.w3.org/2000/svg",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "d": "M17.657 10.333a1.317 1.317 0 0 0-1.1-.6h-2.64V3.511c0-.634-.44-1.18-1.055-1.308a1.317 1.317 0 0 0-1.48.784l-3.96 9.333c-.174.412-.132.885.112 1.26.245.373.66.598 1.103.598h2.64V20.4c-.001.634.44 1.18 1.055 1.308.615.127 1.233-.2 1.48-.784l3.96-9.333c.173-.412.13-.884-.115-1.258zM12.597 20.4v-7.556h-3.96l3.96-9.333v7.556h3.96l-3.96 9.333z",
        "fill": "black",
        "fillRule": "evenodd",
      },
      "ref": null,
      "rendered": null,
      "type": "path",
    },
    "type": "svg",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <path
          d="M17.657 10.333a1.317 1.317 0 0 0-1.1-.6h-2.64V3.511c0-.634-.44-1.18-1.055-1.308a1.317 1.317 0 0 0-1.48.784l-3.96 9.333c-.174.412-.132.885.112 1.26.245.373.66.598 1.103.598h2.64V20.4c-.001.634.44 1.18 1.055 1.308.615.127 1.233-.2 1.48-.784l3.96-9.333c.173-.412.13-.884-.115-1.258zM12.597 20.4v-7.556h-3.96l3.96-9.333v7.556h3.96l-3.96 9.333z"
          fill="black"
          fillRule="evenodd"
        />,
        "height": 20,
        "viewBox": "0 0 24 24",
        "width": 20,
        "xmlns": "http://www.w3.org/2000/svg",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "d": "M17.657 10.333a1.317 1.317 0 0 0-1.1-.6h-2.64V3.511c0-.634-.44-1.18-1.055-1.308a1.317 1.317 0 0 0-1.48.784l-3.96 9.333c-.174.412-.132.885.112 1.26.245.373.66.598 1.103.598h2.64V20.4c-.001.634.44 1.18 1.055 1.308.615.127 1.233-.2 1.48-.784l3.96-9.333c.173-.412.13-.884-.115-1.258zM12.597 20.4v-7.556h-3.96l3.96-9.333v7.556h3.96l-3.96 9.333z",
          "fill": "black",
          "fillRule": "evenodd",
        },
        "ref": null,
        "rendered": null,
        "type": "path",
      },
      "type": "svg",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
